pennylane/templates/subroutines/out_poly.py

```diff
@@ -23,7 +23,7 @@ def _get_polynomial(f, mod, *variable_sizes):
     """Calculate the polynomial binary representation of a given function using the `Möbius inversion formula <https://en.wikipedia.org/wiki/Möbius_inversion_formula#On_posets>`_ .
 
     Args:
-        f (callable):  the function from which the polynomial is extracted
+        f (callable):  the function from which the polynomial is extracted. f is the zeta transform of the extracted polynomial.
```

Comment:
This confuses me a bit. So `f` should implement the zeta transform of the polynomial from which it is extracted? I think you should make this clearer.

Reply:
yes, probably  this clarification confuses more than helps. I eliminate it. f is the poly function that fulfills (although the user does not have to think about it) to be the zeta transform of the coefficients that we are looking for

Reply:
(I remove it @willjmax  since Comment in line 73 clarify that )